packages/components/src/button/style.scss

```diff
@@ -118,21 +118,15 @@
 		outline: 1px solid transparent;
 
 		&:active:not(:disabled) {
-			background: $components-color-gray-300;
-			color: $components-color-accent-darker-10;
```

Comment:
Not that I object but just checking since it wasn't mentioned in the PR description — is it intentional for the text to turn black when clicked?

https://github.com/WordPress/gutenberg/assets/555336/774e524b-5216-488b-b560-3598a1c7449b